File: simple-stack-example-rx/src/main/java/com/zhuinden/stack_rx_example/BackstackService.java
package com.zhuinden.stack_rx_example;

import android.content.Context;

import com.zhuinden.simplestack.Backstack;

/**
 * Created by Owner on 2017. 02. 11..
 */

public class BackstackService {
    public static final String TAG = "simplestack.BACKSTACK";

    public static Backstack getBackstack(Context context) {
        // noinspection ResourceType
        return (Backstack)context.getSystemService(TAG);
    }
}

File: simple-stack-example-basic-kotlin/src/main/kotlin/com/zhuinden/simplestackdemoexample/common/BackstackService.kt
package com.zhuinden.simplestackdemoexample.common

import android.content.Context

import com.zhuinden.simplestack.Backstack

/**
 * Created by Owner on 2017. 01. 31..
 */

object BackstackService {
    val TAG = "BackstackService"

    fun get(context: Context): Backstack {
        return context.getSystemService(TAG) as Backstack
    }
}
